apps/office-addin/src/taskpane/taskpane.html

```diff
@@ -1,26 +1,10 @@
-<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. -->
-<!-- See LICENSE in the project root for license information -->
-
 <!DOCTYPE html>
 <html lang="en" data-framework="typescript">
   <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>KlickerUZH</title>
-
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
no need for google fonts, as the layout can just use arial, and we do not want everyone to connect to g fonts